packages/knip/fixtures/plugins/webpack/webpack.common.js

```diff
@@ -97,6 +97,11 @@ module.exports = () => {
         Vue: ['vue/dist/vue.esm.js', 'default'],
       }),
     ],
+    resolveLoader: {
+      alias: {
+        'my-loader': path.resolve(__dirname, 'src/my-loader.js'),
```

Comment:
Just to be sure, any chance either the alias or the filename could be changed? To distinguish `my-loader` resolving to the local file vs the alias/dependency.

Reply:
absolutely, good idea: c0f284ab

we now have `my-loader` referencing `my-custom-loader.js`:

```
'my-loader': path.resolve(__dirname, 'src/my-custom-loader.js'),
```